srcpkgs/steam/template

```diff
@@ -21,15 +21,20 @@ do_install() {
 	# Ignore steamdeps.
 	ln -sf /bin/true ${DESTDIR}/usr/bin/steamdeps
 
-	# Steam Controller support: need read-write access to
-	# Valve-owned input event device nodes.
-	# See https://bugs.launchpad.net/ubuntu/+source/steam/+bug/1498655
-	vinstall ${FILESDIR}/99-steam-controller-perms.rules 644 usr/lib/udev/rules.d
-
-	# Device support for Steam-related hardware (e.g. controllers over Steam Link)
-	vinstall ${FILESDIR}/60-steam-input.rules 644 usr/lib/udev/rules.d
-	vinstall ${FILESDIR}/60-steam-vr.rules 644 usr/lib/udev/rules.d
-
 	# Void-specific documentation.
 	vdoc "${FILESDIR}/README.voidlinux"
 }
+
+steam-udev-rules_package() {
+	short_desc+=" - udev rules for hardware"
+	pkg_install() {
+		# Steam Controller support: need read-write access to
+		# Valve-owned input event device nodes.
+		# See https://bugs.launchpad.net/ubuntu/+source/steam/+bug/1498655
+		vinstall ${FILESDIR}/99-steam-controller-perms.rules 644 usr/lib/udev/rules.d
+
+		# Device support for Steam-related hardware (e.g. controllers over Steam Link)
+		vinstall ${FILESDIR}/60-steam-input.rules 644 usr/lib/udev/rules.d
+		vinstall ${FILESDIR}/60-steam-vr.rules 644 usr/lib/udev/rules.d
```

Comment:
it's better to `vmove usr/lib/udev/rules.d` in a subpackage definition and leave the `vinstall`s where they were